src/scripts.js

```diff
@@ -98,7 +98,7 @@ function displayInfo(userInfo, hydrationInfo, sleepInfo) {
 
 function displayHydrationInfo(userInfo, id, hydrationInfo) {
   let hydrationRepo = new Hydration(hydrationInfo);
-  let today = makeToday(userInfo, id, hydrationRepo.hydrationData);
+  let today = userInfo.getToday(id, hydrationRepo.hydrationData);
```

Comment:
Nice - this solves the refactor problem we had with makeToday() being called multiple times in scripts